Hello-World.html

```diff
@@ -2,6 +2,12 @@
 <!DOCType html>
   <head>
     <title> Hello World </title>
+    <style>
+      div.header {
+        text-align: center;
+        color: red;
+      }
+    </style>
```

Comment:
think about adding .header definition too. This one will be able to used by any tag that has attribute class="header"